packages/react/react/src/hooks/useCombinedRefs.spec.tsx

```diff
@@ -5,24 +5,26 @@ import { useCombinedRefs } from './useCombinedRefs';
 
 describe('useCombinedRefs', () => {
   it('여러 개의 ref를 하나로 합칠 수 있다.', async () => {
-    const refs: Array<HTMLDivElement | null> = [];
-    const callbackRef = jest.fn();
+    const refs: Array<HTMLDivElement | null | undefined> = [];
+    const refCallback = jest.fn();
 
     function TestComponent() {
-      const ref1 = useRef<HTMLDivElement>(null);
-      const ref2 = useRef<HTMLDivElement>(null);
-      const ref3 = useCallback((element: HTMLDivElement | null) => {
-        callbackRef(element);
+      const ref1 = useRef<HTMLDivElement>();
+      const ref2 = useRef();
+      const ref3 = useRef(null);
+      const ref4 = useRef<HTMLDivElement>(null);
+      const ref5 = useCallback((element: HTMLDivElement | null) => {
+        refCallback(element);
       }, []);
 
-      const ref = useCombinedRefs(ref1, ref2, ref3);
+      const ref = useCombinedRefs(ref1, ref2, ref3, ref4, ref5);
```

Comment:
https://github.com/toss/slash/pull/211#discussion_r1097054749

So please revert this change.